.config/minified.js

```diff
@@ -19,6 +20,9 @@ const minConfig = {
     }
   },
   plugins: baseConfig.plugins.concat([
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('production')
```

Comment:
If I passed `'production'` here, it would end up as `production` in the code, and be treated as an undefined variable. I guess I could've gone with `'"production"'` or something like that, but I always used stringify, so I went with it.